Rakefile

```diff
@@ -43,8 +43,6 @@ require 'solr_wrapper/rake_task'
 desc 'Lint, set up test app, spin up Solr, and run specs'
 task ci: [:rubocop] do
```

Comment:
Do these changes mess up the ability to run `rake ci` on a development machine?

Reply:
Tested this locally it works as long as you don't have an instance of solr_wrapper --config solr_wrapper_test running in another terminal